output/transform/postgres_relations.go

```diff
@@ -119,6 +119,27 @@ func transformPostgresRelations(s snapshot.FullSnapshot, newState state.Persiste
 			}
 			info.Columns = append(info.Columns, &sColumn)
 		}
+
+		if schemaStatsExist {
+			schemaStatsExtended, ok := schemaStats.RelationStatsExtended[relation.Oid]
+			if ok {
+				for _, s := range schemaStatsExtended {
+					sExtendedStats := snapshot.RelationInformation_ExtendedStatistic{
+						StatisticsSchema: s.StatisticsSchema,
+						StatisticsName:   s.StatisticsName,
+						Columns:          s.Columns,
+						Expressions:      s.Expressions,
+						Kind:             s.Kind,
+						HasData:          s.Inherited.Valid || s.NDistinct.Valid || s.Dependencies.Valid, // Only has valid data when either of these are valid
```

Comment:
[note for reviewer] `inherited` is non-null column so it would have been best to identify if we were able to obtain the actual stats data of this extended stats, but `s.Inherited` can be null for version 14 and below, so it can't be simply used. As a workaround, we check all three data values to see if we got any data here.
While things like `columns` above can be obtained without special permissions/privileges, `n_distinct` will only be available with the right permission/privilege (originally coming from [pg_statistic_ext_data](https://www.postgresql.org/docs/current/catalog-pg-statistic-ext-data.html)).